src/api/auth.js

```diff
@@ -5,9 +5,11 @@ import { getBaseUrl, getInnerBaseUrl } from "../common/environment.js";
 
 export const AUTH_API_NAMES = {
   getAuthTokenTestUser: "transactions/getAuthTokenTestUser",
+  getUserInfo: "auth/getUserInfo",
```

Comment:
it's not related to this task, but could you please fix the row above in `auth/getAuthTokenTestUser`